### ISSUE TITLE
Error: Microsoft.Net.Compilers is only supported on MSBuild v16.3 and above

### ISSUE THREAD

**Issue by JesperNoerregaard** [dotnet/roslyn#40130]
**Version Used**: Microsoft.Net.Compilers 3.4.0

**Steps to Reproduce**:

1. Update package Microsoft.Net.Compilers from 3.3.1 to 3.4.0 in project A (.Net Framework 4.8 MVC 5 Web App)
2. Build MSTest project that references project A (.Net Framework 4.8 class library)

**Expected Behavior**: No error message

**Actual Behavior**: Error message "Microsoft.Net.Compilers is only supported on MSBuild v16.3 and above."

It seems that inside "Microsoft.Net.Compilers.Props" the variable $(MSBuildToolsVersion) evaluates to "Current". Hence the condition "'$(MSBuildToolsVersion)' &lt; '16.3'" is not met

**Comment by JesperNoerregaard** [comment]
Oh, this was created minutes after #40129 - sorry for the duplicate

**Comment by erikbra** [comment]
Same issue here. Works on 3.3.1

**Comment by sharwell** [comment]
Duplicate of #40129
